source/documentation/allowances-and-reliefs.html.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Allowances and reliefs | Tax Logic service guide
+weight: 3
+description: Details of the calculations used for allowances and reliefs.
+---
+
+# Allowances and reliefs
+
+Income Tax allowances and reliefs help reduce the amount of tax that customers pay.
+
+## Personal Allowance
+
+For information about Personal Allowance, refer to [Income Tax rates and Personal Allowances (GOV.UK)](https://www.gov.uk/income-tax-rates).
+
+Below is the calculation pseudocode for Personal Allowance.
+
+````
+// Personal Allowances for tax year 2024-25
+
+&nbsp;   // Earnings <= £100,000: Full Personal Allowance (£12,570)
```

Comment:
Thes non-breaking-spaces are being escaped and are appearing as `&nbsp` when previewing the docs